src/spacel/provision/template/vpc.py

```diff
@@ -75,36 +78,43 @@ def vpc(self, orbit, region):
-            nat_eip_clone = deepcopy(base_nat_eip)
-            nat_eip_clone['Condition'] = 'MultiAzNat'
-            nat_eip_resource = 'NatEip%02d' % az_index
-            resources[nat_eip_resource] = nat_eip_clone
-            outputs[nat_eip_resource] = {
-                'Value': {'Fn::If': ['MultiAzNat',
-                                     {'Ref': nat_eip_resource}, '']}
-            }
-
-            # Each AZ _can_ have a NAT gateway:
-            nat_gateway_clone = deepcopy(base_nat_gateway)
-            nat_gateway_clone['Condition'] = 'MultiAzNat'
-            nat_gateway_props = nat_gateway_clone['Properties']
-            nat_gateway_props['SubnetId']['Ref'] = nat_subnet_resource
-            nat_gateway_props['AllocationId']['Fn::GetAtt'][0] = \
-                nat_eip_resource
-            nat_gateway_resource = 'NatGateway%02d' % az_index
-            resources[nat_gateway_resource] = nat_gateway_clone
+            if private_nat_gateway:
+                # Each AZ _can_ have a NAT gateway:
+                nat_eip_clone = deepcopy(base_nat_eip)
+                nat_eip_clone['Condition'] = 'MultiAzNat'
+                nat_eip_resource = 'NatEip%02d' % az_index
+                resources[nat_eip_resource] = nat_eip_clone
+                outputs[nat_eip_resource] = {
+                    'Value': {'Fn::If': ['MultiAzNat',
+                                         {'Ref': nat_eip_resource}, '']}
+                }
+
+                # Each AZ _can_ have a NAT gateway:
+                nat_gateway_clone = deepcopy(base_nat_gateway)
+                nat_gateway_clone['Condition'] = 'MultiAzNat'
+                nat_gateway_props = nat_gateway_clone['Properties']
+                nat_gateway_props['SubnetId']['Ref'] = nat_subnet_resource
+                nat_gateway_props['AllocationId']['Fn::GetAtt'][0] = \
+                    nat_eip_resource
+                nat_gateway_resource = 'NatGateway%02d' % az_index
+                resources[nat_gateway_resource] = nat_gateway_clone
 
             # Each private route table has a default NAT route:
             private_default_route_clone = deepcopy(base_default_route)
             private_route_props = private_default_route_clone['Properties']
             private_route_props['RouteTableId']['Ref'] = private_rt_resource
-            private_route_props['NatGatewayId'] = {
-                'Fn::If': [
-                    'MultiAzNat', {'Ref': nat_gateway_resource},
-                    {'Ref': 'NatGateway01'}
-                ]
-            }
+            if private_nat_gateway:
+                private_route_props['NatGatewayId'] = {
+                    'Fn::If': [
+                        'MultiAzNat', {'Ref': nat_gateway_resource},
+                        {'Ref': 'NatGateway01'}
+                    ]
+                }
+            else:
+                del private_route_props['NatGatewayId']
+                private_route_props['GatewayId'] = {
```

Comment:
Can we just remove the route? Blackholing the internet feels better than having something labelled Private that is actually Public.